tests/graph_index/test_graph_index.py

```diff
@@ -1,175 +0,0 @@
-from dgl import DGLError
```

Comment:
why removing this file?

Reply:
It looks a duplicate of `tests/compute/test_graph_index.py`.